### DISCUSSION TITLE
Possible Macros date/time bug

### DISCUSSION THREAD

**Discussion by gcwong23** [WarmUpTill/SceneSwitcher#560] [General]
Program is very cool… however I noticed with the new updates that when using the date/time macros, the time is 7 hrs off. I set 13:00 for the start record time, then I tab out and back in the macros and it shows 7 hrs later…

**Comment by WarmUpTill** [comment]
I noticed the same recently - seems to be a visual bug.
The values you entered are being used.

A build with a fix is available here:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/3199862118
